pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -53,12 +54,12 @@ import (
 var ctx context.Context
 var env *test.Environment
 var cluster *state.Cluster
-var controller *deprovisioning.Controller
-var provisioningController *provisioning.Controller
+var deprovisioningController *deprovisioning.Controller
```

Comment:
Optional: Since we're already in the package deprovisioning, I'm less compelled by this